kyuubi-server/src/test/scala/org/apache/kyuubi/engine/hive/HiveProcessBuilderSuite.scala

```diff
@@ -20,12 +20,15 @@ package org.apache.kyuubi.engine.hive
 import org.apache.kyuubi.KyuubiFunSuite
 import org.apache.kyuubi.config.KyuubiConf
 import org.apache.kyuubi.config.KyuubiConf.{ENGINE_HIVE_EXTRA_CLASSPATH, ENGINE_HIVE_JAVA_OPTIONS, ENGINE_HIVE_MEMORY}
+import org.apache.kyuubi.engine.hive.HiveProcessBuilder._
 
 class HiveProcessBuilderSuite extends KyuubiFunSuite {
 
   test("hive process builder") {
     val conf = KyuubiConf().set("kyuubi.on", "off")
-    val builder = new HiveProcessBuilder("kyuubi", conf)
+    val builder = new HiveProcessBuilder("kyuubi", conf) {
```

Comment:
why we need this? change

Reply:
because kyuubi-server does not depend on hive-engine anymore

Reply:
before this PR, this test did not rely on hive-engine, so why do we need this change.

Reply:
Before this PR, the hive engine builds before kyuubi-server because kyuubi-server depends on hive engine's test jar.
After this PR, the hive engine builds after kyuubi-server, then the following code will throw exception.
https://github.com/apache/incubator-kyuubi/blob/e3bf6044ec6d381cc5e56aaff1cf0a46592a87e1/kyuubi-server/src/main/scala/org/apache/kyuubi/engine/hive/HiveProcessBuilder.scala#L89-L103

Reply:
so you can set ENGINE_HIVE_EXTRA_CLASSPATH in L28?

Reply:
yea, it's another option. I follow flink engine to use overwrite env, but I'm open to changing both to overwrite conf.

Reply:
thanks for being open, I guess it conforms to the original intent of these tests if we do not use overwrite env